samples/highcharts/website/icon-map/demo.js

```diff
@@ -0,0 +1,685 @@
+                    style: {
+                        fontFamily: 'IBM Plex Sans',
+                        color: '#fff'
+                    }
+                },
+                exporting: {
+                    enabled: false
+                },
+                legend: {
+                    title: {
+                        text: 'Population density per km²'
+                    },
+                    labelStyle: {
+                        color: '#fff'
+
+                    },
+
+                    floating: true,
+                    y: 20
+                },
+                colorAxis: {
+                    min: 1,
+                    max: 1000,
+                    type: 'logarithmic',
+                    maxColor: '#4455f2'
+                },
+                mapNavigation: {
+                    enabled: true,
+                    buttonOptions: {
+                        verticalAlign: 'bottom',
+                        x: 5
+                    }
+                },
+                tooltip: {
+                    useHTML: true,
+                    distance: -15,
+                    formatter: function () {
+                        const htmlString =
+                        `<div class="tip-grid">
+                        <div class="tip-content">
+                            <div class="dot"></div>${this.point.name}
+                        </div>
+                        <i class="fas fa-caret-down tip-point"></i>
+                        </div>
+                        `;
+                        return htmlString;
+                    },
+                    valueSuffix: '/km²'
+                },
+                series: [{
+                    data: data,
+                    mapData: Highcharts.maps['custom/world-highres'],
+                    joinBy: ['iso-a2', 'code'],
+                    name: 'Population density',
+                    allowPointSelect: true,
+                    cursor: 'pointer',
+                    events: {
+                        click: function (e) {
+                            e.point.zoomTo();
+                        }
+                    }
+
+                }],
+                responsive: {
+                    rules: [{
+                        condition: {
+                            maxWidth: 250
+                        },
+                        chartOptions: {
+                            subtitle: {
+                                text: ''
+                            },
+                            chart: {
+                                margin: [40, 1, 65, 0]
+                            }
+                        }
+                    },
+                    {
+                        condition: {
+                            minWidth: 499
+                        },
+                        chartOptions: {
+                            subtitle: {
+                                text: 'Click a country to zoom to it. Use buttons below map for selected tests.'
```

Comment:
Which buttons? The demo doesn't have buttons below the map - maybe the website does?

Reply:
The buttons below the interactive gantt demo on the website (this is the same demo, just styled.) 

I think what is happening is there's a 'styledMode' check script that runs in the utils frame. I'm loading one chart in the utils frame, then wiping it out and loading another chart, and this causes a JS error, which then causes the buttons not to appear.

In the "mobile" view of the chart in utils, everything works fine, because that styledMode checker script doesn't seem to run.
[
<img width="530" alt="image" src="https://user-images.githubusercontent.com/1271445/134415257-3bfab76f-a7ef-4dcd-b41b-5e7ce189ba19.png">
](url)